content/reference/user/simulations/index.md

```diff
@@ -94,12 +94,11 @@ You can add several pools with different numbers of injectors to run your simula
 
 After this step, you can save the simulation, or click on *Next* to access optional configurations.
 
-### Step 3 & 4: JVM options & Injector Parameters
+### Step 3: Injector Parameters
 
-These steps allow you to define the JVM options, Java system properties and environment variables used when running this particular simulation. You can choose to override the global properties.
+This step allows you to define the Java system properties and environment variables used when running this particular simulation. Properties/variables entered here will add to the defaults, unless you choose to ignore the defaults. If you keep the defaults, and you add a property/variable with the same key as one from the defaults, the simulation's value will be used (it overrides the default).
```

Comment:
Tried to clarify what happens here, not sure if it's better?

Reply:
Let's see if i understood well ;) :
 
Give me a try: 

Properties and variables entered here will be added to this particular simulation.
If you previously ask to keep the defaults properties, they will be added to the default one and you have the possibility to override them here.

Reply:
Erf, we do have an UX issue, here.
The same toggle affects two (three?) different set of values.
Perhaps should we add more toggles or merge the different views?

Reply:
I'd rather ship this and have users' feedback than spending more time to have something perfect and unused.